content/en/docs/concepts/containers/container-environment.md

```diff
@@ -52,7 +52,7 @@ FOO_SERVICE_PORT=<the port the service is running on>
 ```
 
 Services have dedicated IP addresses and are available to the Container via DNS,
-if [DNS addon](https://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/) is enabled. 
+if [DNS addon](https://releases.k8s.io/master/cluster/addons/dns/) is enabled. 
```

Comment:
All the updates for https://releases.k8s.io/ URLs would actually be better like this:
```suggestion
if [DNS addon](https://releases.k8s.io/{{< param "fullversion" >}}/cluster/addons/dns/) is enabled.
```

Reply:
I didn't change it because the PR had a lot of changes and was not good to review, so this PR only changed the `githubbranch` parameter.

This parameter `{{< param "fullversion" >}}` also appears many times in the documentation and let me create a new PR for this parameter.

Reply:
Thanks for catch and I will update it.

Reply:
Updated.